docs/guides/api-gateway/kubernetes.mdx

```diff
@@ -0,0 +1,414 @@
+  dashboard](https://dashboard.ngrok.com/get-started/your-authtoken).
+- `{API_KEY}`: An API key, created in the [ngrok
+  dashboard](https://dashboard.ngrok.com/api-keys) to associate with your
+  deployment.
+
+```bash
+export NGROK_AUTHTOKEN={AUTHTOKEN}
+export NGROK_API_KEY={API_KEY}
+```
+
+Install the ngrok Kubernetes Operator into the previously-created `prod` namespace with
+Gateway API functionality enabled.
+
+```bash
+helm install ngrok-ingress-controller ngrok/kubernetes-ingress-controller \
+  --namespace prod \
+  --create-namespace \
+  --set credentials.apiKey=$NGROK_API_KEY \
+  --set credentials.authtoken=$NGROK_AUTHTOKEN \
+  --set useExperimentalGatewayApi=true
+```
+
+## **Step 2**. Configure your cluster to use ngrok as an API gateway {#configure-cluster-api-gateway}
+
+For ngrok to operate as your API gateway, using the native Gateway API, you need
+to configure the following:
+
+- A `GatewayClass` that uses the controller from the ngrok Kubernetes Operator
+  project.
+- A `Gateway` with at least one `listener` attached to the ngrok domain you
+  already reserved.
+- At least one `HTTPRoute`, with your `Gateway` as its parent, that matches
+  paths to one or more existing API services (in any namespace) based on their
+  `name` and `port`.
+
+### Create the API gateway configuration
+
+Create a new YAML file named `ngrok-api-gateway.yaml` and paste in the following
+content.
+
+If you're using the ngrok API service from **Step 0**, all you need to do is
+replace `{YOUR_NGROK_DOMAIN}` with the domain you reserved.
+
+If you're using the example API service, substitute `{SERVICE_NAME}` with
+`ngrok-api-service` and `{PORT}` with `80`. If you're deploying your own API
+service, change those variables accordingly. If you have multiple API services
+in different containers, you'll need to create multiple rules to
+match paths to services.
+
+```yaml
+apiVersion: gateway.networking.k8s.io/v1
+kind: GatewayClass
+metadata:
+  name: ngrok-gatewayclass
+  namespace: prod
+spec:
+  controllerName: ngrok.com/gateway-controller
+---
+apiVersion: gateway.networking.k8s.io/v1
+kind: Gateway
+metadata:
+  name: ngrok-gateway
+  namespace: prod
+spec:
+  gatewayClassName: ngrok-gatewayclass
+  listeners:
+    - name: ngrok-listener
+      #highlight-start
+      hostname: "{YOUR_NGROK_DOMAIN}"
+      #highlight-end
+      port: 443
+      protocol: HTTPS
+      tls:
+        mode: Terminate
+        certificateRefs:
+          - name: ngrokcert
+---
+apiVersion: gateway.networking.k8s.io/v1
+kind: HTTPRoute
+metadata:
+  name: ngrok-route
+  namespace: prod
+spec:
+  parentRefs:
+    - kind: Gateway
+      name: ngrok-gateway
+  hostnames:
+    #highlight-start
+    - "{YOUR_NGROK_DOMAIN}"
+    #highlight-end
+  rules:
+    - matches:
+        - path:
+            type: PathPrefix
+            value: /
+      backendRefs:
+        #highlight-start
+        - name: { SERVICE_NAME }
+          port: { PORT }
```

Comment:
port: {PORT} - Remove spaces

Reply:
The automatic code formatter is doing this, and I don't know how to turn it off. First time it's happened to me, too.